GVRf/Framework/framework/src/main/java/org/gearvrf/io/GearCursorController.java

```diff
@@ -206,7 +203,14 @@ public void onDrawFrame()
                 }
                 thread.initialize();
                 initialized = true;
-                context.getInputManager().activateCursorController(GearCursorController.this);
+                Runnable task = new Runnable()
+                {
+                    public void run()
+                    {
+                        context.getInputManager().activateCursorController(GearCursorController.this);
+                    }
+                };
+                context.runOnTheFrameworkThread(task);
```

Comment:
Why the framework thread? It seems to me that all this adds is some latency. There is no other occasion the cursor controllers are using the framework thread. They probably should be for all callbacks but they don't. Also the callbacks right now are on a number of possible threads. Is this change mandatory for the fix to work?